src/components/Tooltip/Tooltip.js

```diff
@@ -82,16 +82,23 @@ const Tooltip = (props: Props) => {
 
   const opacityAnim = React.useRef<Animated.Value>(new Animated.Value(0)).current;
   React.useEffect(() => {
-    if (tooltipLayout.height) {
+    let mounted = true;
+    if (tooltipLayout.height && mounted) {
       Animated.timing(opacityAnim, {
         toValue: isVisible ? 1 : 0,
         duration: 200,
         useNativeDriver: true,
-      }).start(() => { if (!isVisible) setVisibilityState(false); });
+      }).start(() => {
+        if (!isVisible) setVisibilityState(false);
+      });
     }
-    if (isVisible) {
+    if (isVisible && mounted) {
```

Comment:
Isn't mounted always true here?

Reply:
whenever useEffect cleanup then mounted have false.

Reply:
On line 85 you're setting `mounted` as true, regardless of whether or not you have a clean up function setting it to false - at this stage in the code `mounted` would still be true since there's no delay in between. This is more apparent on line 86, where you use mounted immediately.

Maybe try using a hook instead? https://usehooks-ts.com/react-hook/use-is-mounted